src/components/common/Layout/Header/NetworkDropdown/NetworkDropdown.less

```diff
@@ -31,7 +31,46 @@
     padding-left: calc(var(--spectrum-base-gutter) * 4);
     margin-bottom: var(--spectrum-base-gutter) !important;
   }
+  &__btn--ergo {
```

Comment:
You can write styles only once by usign css-variables. (--network-dropdown-bg, --network-dropdown-border-color and etc.) and then in root style.less create two rules:
`.cardano { --network-dropdown-bg: rgba(68, 103, 191, 0.25)  }` and `.ergo { --network-dropdown-bg: rgba(236, 105, 51, 0.35)` for example